configs/Gigabyte/GA-B550M-AORUS-PRO.conf

```diff
@@ -0,0 +1,32 @@
+# Requires it87 driver with support for IT8688E Super I/O chip
+# Tested with https://github.com/frankcrawford/it87 on B550M AORUS PRO (rev. 1.0)
+
+chip "it8688-isa-0a40"
+
+    label in0 "Vcore"
+    ignore in1 # +3.3V always around 2.0
+    ignore in2 # +12V always around 2.0
+    ignore in3 # +5V always around 2.0
```

Comment:
I've got no clue how to find out correct caltulations or if this is broken in it87 driver completely.

Reply:
These are scaled by resistor dividers, the same as on B550M-DS3H. So this can be dealt with using `compute` statements.
```
    label in1 "+3.3V"
    compute in1 ((6.49/10)+1)*@, @/((6.49/10)+1)
    label in2 "+12V"
    compute in2 ((75/15)+1)*@, @/((75/15)+1)
    label in3 "+5V"
    compute in3 ((15/10)+1)*@, @/((15/10)+1)
```
The coefficients are based on actual resistor values used in the dividers.

Reply:
Thanks for the tip. Actually I just borrowed similar calculations from [this config for another GA B550](https://github.com/lm-sensors/lm-sensors/pull/315/files) (in1 is slighly off compared to your example) and stopped caring since.